kinto/core/resource/schema.py

```diff
@@ -278,6 +289,62 @@ def deserialize(self, cstruct=colander.null):
         return values
 
 
+class CollectionQuerySchema(QuerySchema):
+    """Querystring validated fields used with collections."""
+
+    _limit = QueryField(colander.Integer())
+    _sort = FieldList()
+    _token = QueryField(colander.String())
+    _since = QueryField(colander.Integer())
+    _to = QueryField(colander.Integer())
+    _before = QueryField(colander.Integer())
+    id = QueryField(colander.String())
+    last_modified = QueryField(colander.Integer())
+
+
+class RecordGetQuerySchema(QuerySchema):
+    """Querystring validated fields for GET record requests."""
+
+    _fields = FieldList()
+
+
+class CollectionGetQuerySchema(CollectionQuerySchema):
+    """Querystring validated fields for GET collection requests."""
+
+    _fields = FieldList()
+
+
+# Body Schemas
+
+
+class RecordSchema(colander.MappingSchema):
+
+    data = colander.deferred
```

Comment:
Add a comment to explain that we decide later what to put in data based on the resource schema?